### PR TITLE
Fixed Makefile and cleaned up README

### PR DESCRIPTION
**Pull request by delbonis**
The Makefile didn't work on account of glit's libui dependency being weird, so I tweaked it a bit to make sure that it would at least build *something*.  By default it's just `lit` and `lit-af` for now.

I updated the README to reflect these changes and just cleaned up the "getting started" part as a whole.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,80 +4,79 @@
 
 [![Build Status](http://hubris.media.mit.edu:8080/job/lit-PR/badge/icon)](http://hubris.media.mit.edu:8080/job/lit-PR/)
 
-Under development, not for use with real money.
+Under development, not for use with real money.  Unless you're crazy.
```
Comment (Varunram) [inline README.md]: ?

File: Makefile [hunk 2]
```diff
@@ -1,34 +1,43 @@
...
+lit:
+	build/env.sh go get -v .
+	build/env.sh go build -v
 	@echo "Done building."
 	@echo "Run \"$(GOBIN)/lit\" to launch lit."
 
-lit-af: goget
-	cd cmd/lit-af ; go build
+lit-af:
+	build/env.sh go get -v ./cmd/lit-af
```
Comment (Varunram) [inline Makefile]: The reason `goget` is in place to remove this repeating `go get` thing, would be nice if we could add that back in.

File: Makefile [hunk 3]
```diff
@@ -1,34 +1,43 @@
...
 
-.PHONY: lit lit-af glit test tests
+.PHONY: lit lit-af glit test clean
 
 GOBIN = $(shell pwd)
 GO ?= latest
 WEBUI_REPO = "https://github.com/josephtchung/webui"
-all: lit test
 
+all: lit lit-af
```
Comment (Varunram) [inline Makefile]: all should make `test` as well? That's how `bitcoind`'s make works, so lets do something similar.

**Comment by Varunram** [comment]
Also, build fails due to seem weird reason:
```
3:37:21 ERROR: Build step failed with exception
23:37:21 java.io.FileNotFoundException: https://api.github.com/repos/mit-dci/lit/statuses/17af2a050683528ea4a3768ae2252c89c4c319f4
23:37:21 	at com.squareup.okhttp.internal.huc.HttpURLConnectionImpl.getInputStream(HttpURLConnectionImpl.java:243)
23:37:21 	at com.squareup.okhttp.internal.huc.DelegatingHttpsURLConnection.getInputStream(DelegatingHttpsURLConnection.java:210)
23:37:21 	at com.squareup.okhttp.internal.huc.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:25)
23:37:21 	at org.kohsuke.github.Requester.parse(Requester.java:612)
23:37:21 	at org.kohsuke.github.Requester.parse(Requester.java:594)
23:37:21 	at org.kohsuke.github.Requester._to(Requester.java:272)
23:37:21 Caused: org.kohsuke.github.GHFileNotFoundException: {"message":"Not Found","documentation_url":"https://developer.github.com/v3/repos/statuses/#create-a-status"}
23:37:21 	at org.kohsuke.github.Requester.handleApiError(Requester.java:686)
23:37:21 	at org.kohsuke.github.Requester._to(Requester.java:293)
23:37:21 	at org.kohsuke.github.Requester.to(Requester.java:234)
23:37:21 	at org.kohsuke.github.GHRepository.createCommitStatus(GHRepository.java:1075)
23:37:21 	at org.jenkinsci.plugins.github.status.GitHubCommitStatusSetter.perform(GitHubCommitStatusSetter.java:160)
23:37:21 Caused: org.jenkinsci.plugins.github.common.CombineErrorHandler$ErrorHandlingException
23:37:21 	at org.jenkinsci.plugins.github.common.CombineErrorHandler.handle(CombineErrorHandler.java:74)
23:37:21 	at org.jenkinsci.plugins.github.status.GitHubCommitStatusSetter.perform(GitHubCommitStatusSetter.java:164)
23:37:21 	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:81)
23:37:21 	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
23:37:21 	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:744)
23:37:21 	at hudson.model.AbstractBuild$AbstractBuildExecution.performAllBuildSteps(AbstractBuild.java:690)
23:37:21 	at hudson.model.Build$BuildExecution.post2(Build.java:186)
23:37:21 	at hudson.model.AbstractBuild$AbstractBuildExecution.post(AbstractBuild.java:635)
23:37:21 	at hudson.model.Run.execute(Run.java:1752)
23:37:21 	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
23:37:21 	at hudson.model.ResourceController.execute(ResourceController.java:97)
23:37:21 	at hudson.model.Executor.run(Executor.java:429)
23:37:21 Build step 'Set GitHub commit status (universal)' marked build as failure
23:37:21 Setting status of 38416c69c07f78092f298249d6fdf1ed63251170 to FAILURE with url http://hubris.media.mit.edu:8080/job/lit-PR/145/ and message: 'Build finished. '
23:37:21 Finished: FAILURE
```

File: README.md [hunk 4]
```diff
@@ -4,80 +4,79 @@
...
 
 ## Setup
 
 ### Prerequisites
 
 - [Git](https://git-scm.com/)
-- [Go](https://golang.org/doc/install)
```
Comment (Varunram) [inline README.md]: Oh and go is still a prerequisite.

File: Makefile (hunk 2 above)
Comment (delbonis) [reply Makefile]: I tried to do that but it was complaining about glit still.  I can tweak it though to only download deps for things that actually get built.

**Comment by metalicjames** [comment]
retest this please

**Comment by delbonis** [comment]
retest this please

**Comment by delbonis** [comment]
It'd be useful to update the Jenkinsfile to run `make package` and collect all the archives generated as some of build artifacts.  But that's for another PR.

**Comment by delbonis** [comment]
Should be fine now.

**Pull request merged**